corgi/monitor/consumer.py

```diff
@@ -48,16 +50,44 @@ def __init__(self, virtual_topic_address: str, selector: Optional[str] = None):
         # message that is accepted automatically (this is the default value).
         self.auto_settle = True
 
-    def on_start(self, event: Event):
+    def on_start(self, event: Event) -> None:
+        """Connect to UMB broker(s) and set up a receiver for each virtual topic address"""
         recv_opts = [self.selector] if self.selector is not None else []
         logger.info("Connecting to broker(s): %s", self.urls)
         conn = event.container.connect(urls=self.urls, ssl_domain=self.ssl_domain, heartbeat=500)
-        event.container.create_receiver(
-            conn, self.virtual_topic_address, name=None, options=recv_opts
-        )
+        for virtual_topic_address in self.virtual_topic_addresses:
+            event.container.create_receiver(
+                conn, virtual_topic_address, name=None, options=recv_opts
+            )
```

Comment:
Did you test that this works? The docs around Receivers aren't super clear (and the code is not super readable either) so it's not apparent to me whether a Container supports multiple Receivers over a single Link. My worry here would be that if it only supports one, you're essentially overwriting receivers in a single container until you reach the last one.